### ISSUE TITLE
WeibullAFTFitter.plot_partial_effects_on_outcome

### ISSUE THREAD

**Issue by johndburger** [CamDavidsonPilon/lifelines#1169]
I've used `plot_partial_effects_on_outcome` for both numeric and categorical covariates for the `CoxPHFitter`. I can do this for numeric covariates for `WeibullAFTFitter`, but if I do something like:

```
 aft.plot_partial_effects_on_outcome(covariates='some_categorical_feature', 
    y='cumulative_hazard',
    values=["foo", "bar"])
```
I get `TypeError: must be real number, not numpy.str_`.

Again, this works perfectly for a numeric feature for `WeibullAFTFitter`. Is this only partly implemented for the Weibull model? 

(I'm running version 0.25.4)

**Comment by CamDavidsonPilon** [comment]
hrm, this is a bug I've seen before. It _may_ be fixed in the latest version (0.25.6) 

Yes, looks like I discovered and patched it in 0.25.5: https://github.com/CamDavidsonPilon/lifelines/commit/adf00b0a4ce496fc72752379da784d828a9ccb13

Give that a try and let me know!

**Comment by johndburger** [comment]
Works now, thanks! I should have tried the latest version before I opened a ticket.

Thanks for your quick response, and also thanks for building such a useful package.
